.github/workflows/publish.yml

```diff
@@ -64,3 +64,45 @@ jobs:
           python -m pip install --upgrade pip
           pip install python-doctr
           python -c "import doctr; print(doctr.__version__)"
+
+  conda:
+    if: "!github.event.release.prerelease"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-update-conda: true
+          python-version: 3.8
+      - name: Install dependencies
+        shell: bash -el {0}
+        run: conda install -y conda-build conda-verify anaconda-client
+      - name: Get release tag
+        id: release_tag
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+      - name: Build and publish
+        shell: bash -el {0}
+        env:
+          ANACONDA_API_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
+          VERSION: ${{ steps.release_tag.outputs.VERSION }}
+        run: |
+          echo "BUILD_VERSION=${VERSION}" >> $GITHUB_ENV
+          python setup.py sdist
+          mkdir conda-dist
+          conda build .conda/ -c conda-forge -c pypdfium2-team -c bblanchon --output-folder conda-dist
+          ls -l conda-dist/noarch/*tar.bz2
+          anaconda upload conda-dist/noarch/*tar.bz2
+
+  conda-check:
+    if: "!github.event.release.prerelease"
+    runs-on: ubuntu-latest
+    needs: conda
+    steps:
+      - uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-update-conda: true
+          python-version: 3.8
+      - name: Install package
+        shell: bash -el {0}
+        run: |
+          conda install -c mindee doctr
```

Comment:
@odulcy-mindee can you create a (public) conda channel for mindee and afterwards add the token as `ANACONDA_TOKEN` to the github secrets please ? :sweat_smile: 
@frgfm That's the way how we can create a custom channel (correct me if i m wrong) : https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/create-custom-channels.html by replacing `channel` with `mindee`

Reply:
@odulcy-mindee we need only the token to add :) No first manual upload or something else that's done about the CI with the next release :)